docs/components/best-practices/development/dealing-with-problems-and-exceptions.md

```diff
@@ -12,6 +12,12 @@ tags:
 
 ## Understanding workers
 
+:::caution Camunda Platform 8 only
+The description of workers targets Camunda Platform 8, even if [external tasks in Camunda Platform 7](https://docs.camunda.org/manual/latest/user-guide/process-engine/external-tasks/) are conceptually similar. If you are looking for Camunda Platform 7, visit [understanding Camunda 7 transaction handling](../understanding-transaction-handling-c7/).
+
+Later parts of this best practice are generically applicable though.
```

Comment:
What do you mean by line 18? Do you mean "The remainder of this document is applicable to both Camunda Platform 7 and Camunda Platform 8."?

Are there any other notes we need to add in this doc to make this true, rather than "generally/somewhat applicable"?

Reply:
This is a bit nasty - the first two sections
- [Understanding workers](https://docs.camunda.io/docs/components/best-practices/development/dealing-with-problems-and-exceptions/#understanding-workers)
- [Handling exceptions on a technical level](https://docs.camunda.io/docs/components/best-practices/development/dealing-with-problems-and-exceptions/#handling-exceptions-on-a-technical-level)

Are pretty C8 specific (even if concepts mentioned in both could be applied to C7 too - but that really needs some transfer work in people's heads)-

The other two sections are pretty independent of C7/C8:

- [Handling errors on the process level](https://docs.camunda.io/docs/components/best-practices/development/dealing-with-problems-and-exceptions/#handling-errors-on-the-process-level)
- [Embracing business transactions and eventual consistency](https://docs.camunda.io/docs/components/best-practices/development/dealing-with-problems-and-exceptions/#embracing-business-transactions-and-eventual-consistency)

Open for any good ideas on how to express this @christinaausley